### ISSUE TITLE
ReadableStreamReadResult used in grpc-web-format.d.ts is no longer available type since typescript 4.2

### ISSUE THREAD

**Issue by kskalski** [timostamm/protobuf-ts#248]
Compiling project using grpcweb-transport using latest typescript compiler produces
```
TS2304 error (TS) Cannot find name 'ReadableStreamReadResult'.	
in node_modules\@protobuf-ts\grpcweb-transport\build\types\grpc-web-format.d.ts
```

This is caused by a breaking change in typescript types https://github.com/microsoft/TypeScript/issues/42970
and a fix is to change ``ReadableStreamReadResult`` to ``ReadableStreamDefaultReadResult``

**Comment by timostamm** [comment]
Hey Kamil, 

could you verify with `skipDefaultLibCheck` and - in case it is not sufficient, `skipLibCheck`?

**Comment by kskalski** [comment]
Yes, ``"skipLibCheck": true,`` silences this error (``skipDefaultLibCheck`` doesn't).

**Comment by timostamm** [comment]
Thank you. This seems worth a workaround.

A fix is released in a pre-release version. Can you confirm that `@protobuf-ts/grpcweb-transport@2.2.3-alpha.1` fixes the issue?

**Comment by kskalski** [comment]
Thanks, it fixes the issue.

**Comment by timostamm** [comment]
Fix released in [v2.2.3](https://github.com/timostamm/protobuf-ts/releases/tag/v2.2.3)
